apps/expo-go/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.kt

```diff
@@ -553,46 +575,14 @@ open class ExperienceActivity : BaseExperienceActivity(), StartReactInstanceDele
     }
   }
 
-  fun onEventMainThread(event: ReceivedNotificationEvent) {
-    // TODO(wschurman): investigate removal, this probably is no longer used
-    if (experienceKey != null && event.experienceScopeKey == experienceKey!!.scopeKey) {
-      try {
-        val rctDeviceEventEmitter =
-          RNObject("com.facebook.react.modules.core.DeviceEventManagerModule\$RCTDeviceEventEmitter")
-        rctDeviceEventEmitter.loadVersion(detachSdkVersion!!)
-        reactInstanceManager.callRecursive("getCurrentReactContext")!!
-          .callRecursive("getJSModule", rctDeviceEventEmitter.rnClass())!!
-          .call("emit", "Exponent.notification", event.toWriteableMap(detachSdkVersion, "received"))
-      } catch (e: Throwable) {
-        EXL.e(TAG, e)
-      }
-    }
-  }
-
-  fun handleOptions(options: ExperienceOptions) {
+  private fun handleOptions(options: ExperienceOptions) {
     try {
       val uri = options.uri
       if (uri !== null) {
         handleUri(uri)
-        val rctDeviceEventEmitter =
-          RNObject("com.facebook.react.modules.core.DeviceEventManagerModule\$RCTDeviceEventEmitter")
-        rctDeviceEventEmitter.loadVersion(detachSdkVersion!!)
-        reactInstanceManager.callRecursive("getCurrentReactContext")!!
-          .callRecursive("getJSModule", rctDeviceEventEmitter.rnClass())!!
-          .call("emit", "Exponent.openUri", uri)
-        BranchManager.handleLink(this, uri, detachSdkVersion)
-      }
-      if ((options.notification != null || options.notificationObject != null) && detachSdkVersion != null) {
-        val rctDeviceEventEmitter =
-          RNObject("com.facebook.react.modules.core.DeviceEventManagerModule\$RCTDeviceEventEmitter")
-        rctDeviceEventEmitter.loadVersion(detachSdkVersion!!)
-        reactInstanceManager.callRecursive("getCurrentReactContext")!!
-          .callRecursive("getJSModule", rctDeviceEventEmitter.rnClass())!!
-          .call(
-            "emit",
-            "Exponent.notification",
-            options.notificationObject!!.toWriteableMap(detachSdkVersion, "selected")
```

Comment:
is that intentional to remove this notification stuffs in expo-go?

Reply:
Yes, @vonovak is removing it.